ecs-cluster-infrastructure.tf

```diff
@@ -0,0 +1,277 @@
+resource "aws_ecs_cluster" "infrastructure" {
+  count = local.enable_infrastructure_ecs_cluster ? 1 : 0
+
+  name = local.infrastructure_ecs_cluster_name
+
+  setting {
+    name  = "containerInsights"
+    value = "enabled"
+  }
+}
+
+resource "aws_security_group" "infrastructure_ecs_cluster_container_instances" {
+  count = local.enable_infrastructure_ecs_cluster ? 1 : 0
+
+  name        = "${local.resource_prefix}-infrastructure-ecs-cluster-container-instances"
+  description = "Infrastructure ECS cluster container instances"
+  vpc_id      = aws_vpc.infrastructure[0].id
+}
+
+resource "aws_security_group_rule" "infrastructure_ecs_cluster_container_instances_ingress_tcp" {
+  count = local.enable_infrastructure_ecs_cluster ? 1 : 0
+
+  description       = "Allow container port tcp ingress from public subnet (TO BE CHANGED TO ONLY ALLOW ALB)"
```

Comment:
Is to be changed in the future? Or is this a missed change?

Reply:
Yep this should be changed when we have the loadbalancer for the services created - It should then only allow traffic to be received from the ALB

Reply:
Can we use a pattern for identifying stuff like this? In PHP I used ["@todo"](https://docs.phpdoc.org/guide/references/phpdoc/tags/todo.html#todo) comments. 

It would mean all these "to do later" items can be grouped with a grepable keyword too.

An added benefit is that the description stays as it should, and developmental comments are left in without affecting the resulting resources

Reply:
Yep I'll add some `TODO:` comments 👍 
I'd like it to stay in the description too though, as it shows what it is doing, and hints as to why it's doing the (kind of) wrong thing - whilst looking through resources in the console